src/Components/Common/MonitorAssetPopover.tsx

```diff
@@ -51,13 +53,17 @@ const VitalsMonitorAssetPopover = ({
               <p>{asset?.name}</p>
             </div>
             <div className="flex flex-col gap-1">
-              <p className="text-sm md:text-base">Middleware Hostname:</p>
+              <p className="text-sm text-black md:text-base">
+                Middleware Hostname:
+              </p>
```

Comment:
Can we add this to locale and use `useTranslations`?

Reply:
We use crowdin for translations for other languages, so usually in PR's we only add for english and crowdin should take care of other languages. 

cc: @bodhish